### PR TITLE
Holiday timeseries

### PR DESCRIPTION
**Pull request by endrjuskr**

### CODE REVIEW

**Comment by endrjuskr** [comment]
If you have other libs than `holiday` in mind, please let me know. `holiday` is still supported (last commit in April) and a quite extensive set of available countries.

File: u8timeseries/utils/timeseries_generation.py [hunk 1]
```diff
@@ -199,14 +200,25 @@ def random_walk_timeseries(length: int = 10,
...
+                       prov: str = None,
+                       state: str = None,
+                       length: int = 10,
+                       start_ts: pd.Timestamp = pd.Timestamp('2000-01-01')) -> TimeSeries:
     """
-    Creates a binary TimeSeries that equals 1 at every index that corresponds to a US holiday,
+    Creates a binary TimeSeries that equals 1 at every index that corresponds to selected country's holiday,
     and 0 otherwise. The frequency of the TimeSeries is daily.
 
+    Available countries can be found at https://github.com/dr-prodigy/python-holidays#available-countries
```
Comment (hrzn) [inline u8timeseries/utils/timeseries_generation.py:212]: In docstrings you can write links like this:
```
`click here <https://github.com/dr-prodigy/python-holidays#available-countries>`_
```

File: u8timeseries/utils/timeseries_generation.py [hunk 2]
```diff
@@ -215,11 +227,11 @@ def us_holiday_timeseries(length: int = 10,
     Returns
     -------
     TimeSeries
-        Binary TimeSeries for US holidays.
+        Binary TimeSeries for country's holidays.
     """
 
     times = pd.date_range(periods=length, freq='D', start=start_ts)
```
Comment (hrzn) [inline u8timeseries/utils/timeseries_generation.py:233]: Could we find a way to have also finer granularities than `D` ? E.g. hourly and minutes?

File: u8timeseries/utils/timeseries_generation.py [hunk 3]
```diff
@@ -215,11 +227,11 @@ def us_holiday_timeseries(length: int = 10,
     Returns
     -------
     TimeSeries
-        Binary TimeSeries for US holidays.
+        Binary TimeSeries for country's holidays.
```
Comment (hrzn) [inline u8timeseries/utils/timeseries_generation.py:230]: I would add a comment saying that the method returns (at the moment) a daily time series.

File: u8timeseries/utils/timeseries_generation.py (hunk 2 above)
Comment (hrzn) [reply u8timeseries/utils/timeseries_generation.py:233]: That can come in a future PR though, I guess

**Pull request merged**